METRICS.md

```diff
@@ -77,6 +77,20 @@ _All metrics start with `eradius` prefix and the prefix is not included into tab
 |  client_unknown_type_request_total                 | [$NAME, $IP, $PORT, $CNAME, $CIP]     | counter   |
 |  client_bad_authenticator_request_total            | [$NAME, $IP, $PORT, $CNAME, $CIP]     | counter   |
 
+Besides these metrics RADIUS client also may create optional server status metrics. These
+metrics represents active/inactive state of upstream RADIUS servers that RADIUS clients
+send requests to.
+
+RADIUS status metrics are not created at RADIUS startup but should be enabled explicitly
+via `eradius_client:enable_server_status_metrics` API.
+
+If RADIUS server status provides following metric:
+
+| Metric                                             | Labels                                | Type      |
+| ---------------------------------------------------|---------------------------------------| ----------|
+|  client_requests_total                             | [ $IP, $PORT]     | boolean  |
```

Comment:
looks like should be `server_status` name

Reply:
yes, thanks, that is good catch, fixed this too